### ISSUE TITLE
args[property].trim() is not a function

### ISSUE THREAD

**Issue by longhorn09** [longhorn09/lorebot#32]
```
2019-04-17 15:49:29 : DTeam                          !query affects=casting level by 3&int by 1
 47 /home/ntang/lorebot/lorebot.js:1154
 48                 if (args[property].trim().indexOf(' by ') > 0) {       // !query affects=damroll by 2,hitroll by 2
 49                                    ^
 50
 51 TypeError: args[property].trim is not a function
 52     at ProcessQuery (/home/ntang/lorebot/lorebot.js:1154:36)
 53     at Client.client.on (/home/ntang/lorebot/lorebot.js:1552:9)
 54     at Client.emit (events.js:160:13)
 55     at MessageCreateHandler.handle (/home/ntang/lorebot/node_modules/discord.js/src/client/websocket/packets/handler    s/MessageCreate.js:9:34)
 56     at WebSocketPacketManager.handle (/home/ntang/lorebot/node_modules/discord.js/src/client/websocket/packets/WebSo    cketPacketManager.js:103:65)
 57     at WebSocketConnection.onPacket (/home/ntang/lorebot/node_modules/discord.js/src/client/websocket/WebSocketConne    ction.js:330:35)
 58     at WebSocketConnection.onMessage (/home/ntang/lorebot/node_modules/discord.js/src/client/websocket/WebSocketConn    ection.js:293:17)
 59     at WebSocket.onMessage (/home/ntang/lorebot/node_modules/ws/lib/event-target.js:120:16)
 60     at WebSocket.emit (events.js:160:13)
 61     at Receiver._receiver.onmessage (/home/ntang/lorebot/node_modules/ws/lib/websocket.js:145:47)
 62 error: Forever detected script exited with code: 1
 63 error: Script restart attempt #1
 64 Lorebot ready!
 65 2019-04-17 15:50:57 : DTeam                          !query affects=casting level by 1,int by 1
```

**Comment by longhorn09** [comment]
Have you tested perhaps converting to string - error suggests to me it might be a casting issue. Like first cast the object to a string, then maybe you could use the trim().

`args[property].toString().trim()....`

**Comment by coloradoderek** [comment]
> Have you tested perhaps converting to string - error suggests to me it might be a casting issue. Like first cast the object to a string, then maybe you could use the trim().
> 
> `args[property].toString().trim()....`

I did think of that and when I ran the test I received the same error, only instead of .trim it said .toString is the invalid function suggesting (in my opinion) that args[property] is where the issue lies.

If you look at args[property] when you do affects=affect1,affect2 versus affects=affect1&affects=affect2, it is different (output from querystring) and that difference is what I think is causing the problem but I'm unsure why or how to resolve it.  easier to remove the unnecessary .functions.
